lib/helium/collection.rb

```diff
@@ -80,6 +80,36 @@ def last
       collection.last
     end
 
+    # Adds resources of the same type to the collection related to the
+    # '@belongs_to' resource
+    def add_relationships(items)
+      body = relationship_request_body(items)
+
+      @client.post(relationship_path, body: body) if relationship_path
+
+      self
```

Comment:
after the post, will the `Collection` instance be updated? like could I chain `label.sensors.add_relationships(new_sensors).each{|sensor| puts sensor}` and it would show all sensors including the newly added ones?